66. Plus One, 30 min.py

```diff
@@ -0,0 +1,12 @@
+class Solution:
+    def plusOne(self, digits):
+        if len(digits) == 1 and digits[0] == 9:
+            return [1, 0]
+
+        if digits[-1] != 9:
+            digits[-1] += 1
+            return digits
+        else:
+            digits[-1] = 0
+            digits[:-1] = self.plusOne(digits[:-1])

```

Comment:
Try practice without recursion and array assignments.

Recursion hides complexity and difficult to analyze.

Array assignments does not transfer to Java or C++ easily.